app-games/src/main/java/com/aptoide/android/aptoidegames/appview/AppViewScreen.kt

```diff
@@ -397,17 +400,112 @@ fun DetailsView(app: App) {
     modifier = Modifier.padding(top = 16.dp)
   ) {
     app.screenshots?.let { ScreenshotsList(it) }
+    app.news?.let {
+      WhatsNew(app = app)
+    }
+
     app.description?.let {
       Text(
         text = it,
-        modifier = Modifier.padding(top = 24.dp, bottom = 32.dp, start = 16.dp, end = 16.dp),
+        modifier = Modifier.padding(top = 16.dp, bottom = 32.dp, start = 16.dp, end = 16.dp),
         style = AGTypography.ArticleText,
         color = Palette.White
       )
     }
   }
 }
 
+@Composable
+fun LatestBox() {
+  Box(
+    modifier = Modifier
+      .height(24.dp)
+      .width(68.dp)
+      .background(Palette.GreyLight),
+  ) {
+    Row {
+      Icon(
+        modifier = Modifier
+          .padding(start = 3.dp, end = 4.dp, top = 4.dp, bottom = 4.dp)
+          .size(16.dp),
+        imageVector = getBookmarkStar(Palette.Black),
+        contentDescription = "bookmark star",
+        tint = Color.Unspecified
+      )
+      Text(
+        modifier = Modifier
+          .padding(end = 8.dp),
+        text = stringResource(id = R.string.latest_title),
+        color = Palette.Black,
+        style = AGTypography.InputsS,
+        textAlign = TextAlign.Center,
+      )
+    }
+
+  }
+}
+
+@Composable
+fun WhatsNew(app: App) {
+  Box(
+    modifier = Modifier
+      .fillMaxWidth()
+      .padding(start = 16.dp, end = 16.dp, top = 24.dp, bottom = 16.dp)
+      .background(Palette.GreyDark)
+  ) {
+    Column(
+      horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+      LatestBox()
+      Row(
```

Comment:
To fully match the design, the last text could be aligned to be centered vertically and the first two texts could be aligned by their baselines (with the `alignByBaseline` modifier)